scripts/qemu-dm-wrapper

```diff
@@ -104,10 +109,21 @@ def main(argv):
     if os.path.exists("/usr/share/qemu-linaro/qemu"):
         qemu_args = qemu_args + ["-L", "/usr/share/qemu-linaro/qemu"]
 
+    if xenstore_exists("/local/domain/%d/platform/nic_type" % domid):
+        print "No explicit NIC selected. Defaulting to rtl8139. "
+    else:
+        nic = xenstore_read("/local/domain/%d/platform/nic_type" % domid)
+        if nic != None:
+            print "qemu: nic_type %s" % nic
+            qemu_args = map(lambda x: x.replace("rtl8139", "%s" % nic), qemu_args)
```

Comment:
You need to take care not to replace anything outside of a network configuration option.  This will currently clobber any `rtl8139` anywhere on the command line, including in logfile paths, etc.